### DISCUSSION TITLE
hi there we are considering switching to [Aug 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6198] [Discord Threads]
> 🙋‍♂️ **alex123452111**

**Comment by o1lab** [comment]
> 🙋‍♂️ **arturvisualsitemaps**  
#### what kind of implications does this have when 2 users are editing the same records? how does this manifest? does it show that "*User-1 is currently editing*"  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1127944168740757504)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
